schemas/jsonldIdentifier.js

```diff
@@ -0,0 +1,26 @@
+/*!
```

Comment:
I recommend we remove this and just use the internals in the two places we need it right now. It does not feel like the right place to draw an abstraction line -- and we shouldn't draw one (certainly not a publicly accessible one) just so that we don't repeat ourselves.